gbpservice/nfp/configurator/api/v1/controllers/controller.py

```diff
@@ -134,7 +134,7 @@ def post(self, **body):
                 body = pecan.request.json_body
 
             if self.method_name == 'network_function_event':
-                routing_key = 'EVENT'
+                routing_key = 'VISIBILITY'
```

Comment:
routing_key change should be done for PUT requests and also the comments block of _entry_to_rpc_routing_table() should be updated.